lib/lte_link_control/lte_lc.c

```diff
@@ -928,10 +928,12 @@ int lte_lc_psm_get(int *tau, int *active_time)
 		return -EBADMSG;
 	}
 
-	/* It's ok not to have legacy Periodic-TAU, older FWs don't provide it. */
 	comma_ptr = strchr(comma_ptr + 1, ch);
 	if (comma_ptr) {
 		strncpy(tau_legacy_str, comma_ptr + 2, 8);
+	} else {
+		LOG_ERR("AT command parsing failed");
```

Comment:
Also here I'm a little bit worried about flash consumption. If the string is the same, it will only be stored into flash once. But if each branch has its own error message, all of those need to be stored into flash. This wouldn't be such a big deal for example in modem_shell, but with LTE LC I think this is something we should take into account.

Reply:
I did not change the strings because that would have increased flash consumption. LTE LC logging already consumes quite a lot of flash.